docs/en/user/configuration.rst

```diff
@@ -68,7 +68,9 @@ Two factor authentication (2FA)
     Two-factor authentication (also known as 2FA) is a technology patented in 1984
     that provides identification of users by means of the combination of two different components.
 
-https://en.wikipedia.org/wiki/Two-factor_authentication
+    https://en.wikipedia.org/wiki/Two-factor_authentication
+    
+**Warning**: enabling 2FA from the configuration interface is only possible if it has been authorized before in `app/config/parameters.yml` by setting the *twofactor_auth* property to `true` (do not forget to empty `/var/cache` after modification).
```

Comment:
Could you please avoid saying to clear the folder?
Instead it's more safe to use `php bin/console cache:clear -e=prod`